### PR TITLE
[1LP][RFR] Removed unneeded VersionPicks

### PR DESCRIPTION
**Pull request by quarckster**
Intent
=================
- `widgetastic.utils.VersionPick` replaced by `cfme.utils.VersionPicker`
- removed unneeded version picks

{{pytest: -v -k "test_automate_method_inputs_crud or test_host_good_creds or test_infra_provider_crud" --long-running}}

### CODE REVIEW

File: cfme/containers/provider/__init__.py [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 from random import sample
-from six import string_types
 from traceback import format_exc
 
-
-import re
+import attr
 from navmazing import NavigateToSibling, NavigateToAttribute
+from six import string_types
 from widgetastic_manageiq import StatusBox, ContainerSummaryTable
```
Comment (sshveta) [inline cfme/containers/provider/__init__.py]: Required : Line 9 and 11 can be merged.

from widgetastic_manageiq import Accordion, ContainerSummaryTable, LineChart, ManageIQTree, StatusBox, SummaryTable

File: cfme/ansible/credentials.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import attr
 from navmazing import NavigateToAttribute, NavigateToSibling
 from widgetastic.exceptions import NoSuchElementException
-from widgetastic.utils import ParametrizedLocator, VersionPick
+from widgetastic.utils import ParametrizedLocator
 from widgetastic.widget import ConditionalSwitchableView, ParametrizedView, Text, TextInput, View
 from widgetastic_patternfly import BootstrapSelect, Button, Dropdown, Input
 from widgetastic_manageiq import ParametrizedSummaryTable, Table, PaginationPane
```
Comment (sshveta) [inline cfme/ansible/credentials.py]: Please sort imports .
from widgetastic_manageiq import ParametrizedSummaryTable, Table, PaginationPane
from widgetastic_patternfly import BootstrapSelect, Button, Dropdown, Input
from widgetastic.utils import ParametrizedLocator
from widgetastic.widget import ConditionalSwitchableView, ParametrizedView, Text, TextInput, View

File: cfme/common/provider_views.py [hunk 3]
```diff
@@ -2,7 +2,6 @@
 from lxml.html import document_fromstring
 
 from widgetastic.exceptions import NoSuchElementException
-from widgetastic.utils import VersionPick, Version
 from widgetastic.widget import View, Text, ConditionalSwitchableView, ParametrizedView
 from widgetastic_patternfly import BreadCrumb, Dropdown, BootstrapSelect, Tab
```
Comment (sshveta) [inline cfme/common/provider_views.py]: Sort imports here too please

File: cfme/common/physical_switch_views.py [hunk 4]
```diff
@@ -19,10 +15,7 @@
     TimelinesView,
     ManageIQTree
 )
-from widgetastic_patternfly import (
-    Dropdown,
-    Accordion
-)
+from widgetastic_patternfly import Dropdown, Accordion
```
Comment (sshveta) [inline cfme/common/physical_switch_views.py]: Sort imports

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/common/physical_switch_views.py (hunk 4 above)
Comment (quarckster) [reply cfme/common/physical_switch_views.py]: Done

**Comment by quarckster** [comment]
Travis failed because it uses DummyAppliance that has version 5.8.

**Comment by mshriver** [comment]
@quarckster you'll need to address the DummyAppliance default to resolve the travis failures.

**Comment by mshriver** [comment]
The travis doesn't specify `--dummy-appliance-version`, and I think this should be fixed with a change to the default version within the `DummyAppliance` class:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/__init__.py#L2983

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2a966f8f928fb2f4689e3b2ab092a40a644ece8d:

`cfme/containers/pod.py`:
- :red_circle: Line 4:1: **F401** *'widgetastic_manageiq.SummaryTable' imported but unused*

`cfme/containers/provider/__init__.py`:
- :red_circle: Line 9:1: **F401** *'widgetastic_manageiq.ContainerSummaryTable' imported but unused*
- :red_circle: Line 9:1: **F401** *'widgetastic_manageiq.SummaryTable' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**